### PR TITLE
Replacing rollbar_access_token_names variable

### PR DESCRIPTION
**Pull request by stoyanzhekov**

### CODE REVIEW

**Comment by Fodoj** [comment]
You also need to adjust tests in this file - https://github.com/babbel/terraform-aws-secretsmanager-for-rollbar-access-tokens/blob/main/_test/main.tf. Please also update PR title and description to reflect the contents

File: _test/main.tf [hunk 1]
```diff
@@ -2,13 +2,26 @@ provider "aws" {
...
+}
+
 module "secretsmanager" {
   source = "./.."
 
   name_prefix = "example"
 
-  rollbar_project_name       = "example"
-  rollbar_access_token_names = ["post_server_item"]
+  rollbar_project_name = "example"
```
Comment (Fodoj) [inline _test/main.tf:23]: This variable is not used anymore and can be dropped

**Comment by jtsaito** [comment]
The [CHANGELOG](https://github.com/babbel/terraform-aws-secretsmanager-for-rollbar-access-tokens/blob/main/CHANGELOG.md) should be updated reflecting this breaking change.

File: _test/main.tf [hunk 2]
```diff
@@ -2,13 +2,25 @@ provider "aws" {
...
+}
+
 module "secretsmanager" {
   source = "./.."
 
   name_prefix = "example"
 
-  rollbar_project_name       = "example"
-  rollbar_access_token_names = ["post_server_item"]
+  rollbar_tokens       = local.rollbar_tokens
```
Comment (jtsaito) [inline _test/main.tf:23]:
```suggestion
  rollbar_tokens = local.rollbar_tokens
```

**Review by jtsaito — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ module "secretsmanager-for-rollbar-access-tokens" {
   name_prefix = "example"
 
   rollbar_project_name       = "example"
-  rollbar_access_token_names = ["post_server_item"]
+  rollbar_tokens             = values(rollbar_project_access_token.example)[*]
```
Comment (jansiwy) [inline README.md:17]: Trailing `[*]` does - AFAIK - nothing:

```suggestion
  rollbar_tokens             = values(rollbar_project_access_token.example)
```

**Review by jansiwy — APPROVED** [review]
apart from that: LGTM!

**Pull request merged**